lib/BC/PcodeCFG.cpp

```diff
@@ -0,0 +1,166 @@
+#include <remill/BC/PCodeCFG.h>
+
+#include <algorithm>
+#include <cstddef>
+#include <sleigh/op.hh>
+#include <sleigh/opcodes.hh>
+#include <sleigh/pcoderaw.hh>
+#include <unordered_map>
+#include <variant>
+#include <vector>
+
+#include "lib/Arch/Sleigh/ControlFlowStructuring.h"
+
+namespace remill {
+namespace sleigh {
+
+PcodeCFG
+PcodeCFGBuilder::CreateCFG(const std::vector<RemillPcodeOp> &linear_ops) {
+  return PcodeCFGBuilder(linear_ops).Build();
+}
+
+
+std::vector<size_t> PcodeCFGBuilder::GetBlockStarts() {
+  std::vector<size_t> res;
+  res.push_back(0);
+
+
+  for (size_t curr_index = 0; curr_index < this->linear_ops.size();
```

Comment:
No need to use `this->`.

Reply:
[ed9069f](https://github.com/lifting-bits/remill/pull/645/commits/ed9069fac284272cf1778130ec1d4659dc80960c)